service/migration_manager.cc

```diff
@@ -746,7 +748,24 @@ future<std::vector<mutation>> prepare_column_family_update_announcement(storage_
         }
         co_await seastar::async([&] {
             db.get_notifier().before_update_column_family(*cfm, *old_schema, mutations, ts);
+
+            // Notify drop indices
+            auto diff = difference(old_schema->all_indices(), cfm->all_indices());
+
+            // indices that are no longer needed
+            for (auto&& name : diff.entries_only_on_left) {
+                const index_metadata& index = old_schema->all_indices().at(name);
+                schema_ptr view;
+                try {
+                    view = db.find_schema(old_schema->ks_name(), name + "_index");
```

Comment:
Please consider using the index_table_name() function instead of open-coding the `+ "_index"`.

Reply:
sure

Reply:
defined one in cql-pytest/utils.py and used it in test_tablets

Reply:
Didn't you change this code to use index_table_name()?
Note that my comment was about the C code, **not** about the test...

Reply:
Fixed in v4